health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/MyBillsTable.js

```
@@ -0,0 +1,168 @@
+                    {
+                        code: "HCM_AM_PDF",
+                        name: t(`HCM_AM_PDF`),
+                    },
+                ]}
+                optionsKey="name"
+                size=""
+                // style={{ minWidth: "15rem" }}
+                title=""
+                type="actionButton"
+                variation="secondary"
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)